pkg/commands/loaders/branches.go

```diff
@@ -31,14 +31,14 @@ type BranchLoaderConfigCommands interface {
 type BranchLoader struct {
 	*common.Common
 	getRawBranches       func() (string, error)
-	getCurrentBranchName func() (string, string, error)
+	getCurrentBranchName func() (string, string, bool, error)
```

Comment:
this function returns enough values that we should have it return a struct with those values as fields. That will make it clearer at the call site what's being returned. And then we can call it getCurrentBranchInfo

Reply:
done